src/helpers/helpers.ts

```diff
@@ -62,13 +62,29 @@ export async function copyFileOrDir(
   sourceFileOrDir: string,
   dest: string,
 ): Promise<void> {
-  return new Promise((resolve, reject) => {
-    ncp(sourceFileOrDir, dest, (error: Error[] | null): void => {
-      if (error) {
-        reject(error);
-      }
-      resolve();
-    });
+  return fs_extra.copy(sourceFileOrDir, dest);
+}
+
+export async function makeFileTreeWriteable(fileOrDir: string): Promise<void> {
+  const mode = 0o755; // read & execute for anyone, plus write for owner
+
+  return fs_extra.lstat(fileOrDir).then((stats) => {
+    if (stats.isFile()) {
+      return fs_extra.chmod(fileOrDir, mode);
+    } else if (stats.isDirectory()) {
+      return fs_extra
+        .readdir(fileOrDir, { withFileTypes: true })
+        .then((children) =>
+          Promise.all(
+            children.map((child) =>
+              makeFileTreeWriteable(path.resolve(fileOrDir, child.name)),
+            ),
+          ),
+        )
+        .then(() => fs_extra.chmod(fileOrDir, mode));
+    } else {
+      return Promise.resolve();
+    }
```

Comment:
Unless I'm missing something, there's nothing to win by using raw promises here. Can you de-promisify this block and replace it with more readable `async` stuff?